docs/05-technical-reference/svls-08-function-specification.md

```diff
@@ -196,6 +202,46 @@ module.exports = {
 }
 ```
 
+</details>
+<details>
+<summary label="Python">
+Python
+</summary>
+
+By default, a failing Function simply throws an error to tell the Event Service to reinject the event at a later point. Such an HTTP-based Function returns the HTTP status code `500`. On the contrary, if you manage to invoke a Function successfully, the system returns the default HTTP status code `200`.
```

Comment:
I wouldn't use `On the contrary` as it expresses strong disagreement with what has just been said, which is not the case here. How about: `In case you manage to invoke a Function successfully (...)`?

Reply:
Thanks for reviewing it @IwonaLanger. I've improved the wording.